### ISSUE TITLE
Bug: New popover commands do not respect the default mode setting

### ISSUE THREAD

**Issue by nothingislost** [nothingislost/obsidian-hover-editor#60]
When using any of the commands to open a new popover, they all open in edit mode regardless of the "default mode" setting.

**Comment by pjeby** [comment]
They follow the setting in the Editor tab, though?

**Comment by nothingislost** [comment]
Yeah, I'm not sure yet how to best respect these two different settings. The reporter wanted to open a file in a new popover and expected it to respect the default hover editor mode. It's a fair assumption but I'm sure some people would expect/want the opposite.
